src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAwaitForeachTests.cs

```diff
@@ -2276,9 +3231,81 @@ public async Task<bool> MoveNextAsync()
                         }
                     }
                 }
-                """ + AsyncStreamsTypes;
-            var comp = CreateCompilationWithTasksExtensions(source, options: TestOptions.ReleaseExe);
+                """;
+            var comp = CreateCompilationWithTasksExtensions([source, AsyncStreamsTypes], options: TestOptions.ReleaseExe);
             CompileAndVerify(comp, expectedOutput: "2 4 -1 Done").VerifyDiagnostics();
+
+            var runtimeAsyncComp = CodeGenAsyncTests.CreateRuntimeAsyncCompilation(source.Replace(" + AsyncStreamsTypes", ""));
```

Comment:
It looks like this Replace call is trying to replace a substring that is not present in `source`